src/content/en/updates/2017/09/sensors-for-the-web.md

```diff
@@ -0,0 +1,410 @@
+ is a fusion sensor that measures rotation of a device in relation to the
+Earth’s coordinate system, while the
+[RelativeOrientationSensor](https://w3c.github.io/orientation-sensor/#relativeorientationsensor-interface)
+ provides data representing rotation of a device hosting motion sensors in
+relation to a stationary reference coordinate system.
+
+All modern 3D JavaScript frameworks support quaternions and rotation matrices
+to represent rotation, however, if you use WebGL directly,
+[OrientationSensor](https://w3c.github.io/orientation-sensor/#orientationsensor-populatematrix)
+ interface has convenient methods for WebGL compatible rotation matrices.
+Here are few snippets:
+
+<div class="clearfix"></div>
+
+**[three.js](https://threejs.org/docs/index.html#api/core/Object3D.quaternion)**
+
+    let torusGeometry = new THREE.TorusGeometry(7, 1.6, 4, 3, 6.3);
+    let material = new THREE.MeshBasicMaterial({ color: 0x0071C5 });
+    let torus = new THREE.Mesh(torusGeometry, material);
+    scene.add(torus);
+
+    // Update mech rotation using quaternion.
+    const sensorAbs = new AbsoluteOrientationSensor();
+    sensorAbs.onreading = () => torus.quaternion.fromArray(sensorAbs.quaternion);
+    sensorAbs.start();
+
+    // Update mech rotation using rotation matrix.
+    const sensorRel = new RelativeOrientationSensor();
+    let rotationMatrix = new Float32Array(16);
+    sensor_rel.onreading = () => {
+        sensorRel.populateMatrix(rotationMatrix);
+        torus.matrix.fromArray(rotationMatrix);
+    }
+    sensorRel.start();
+
+**[BABYLON](http://doc.babylonjs.com/classes/3.0/abstractmesh#rotationquaternion-quaternion-classes-3-0-quaternion-)**
+
+    const mesh = new BABYLON.Mesh.CreateCylinder("mesh", 0.9, 0.3, 0.6, 9, 1, scene);
+    const sensorRel = new RelativeOrientationSensor({frequency: 30});
+    sensorRel.onreading = () => mesh.rotationQuaternion.FromArray(sensorRel.quaternion);
+    sensorRel.start();
+
+**[WebGL](https://www.khronos.org/registry/webgl/specs/latest/2.0/)**
+
+    // Initialize sensor and update model matrix when new reading is available.
+    let modMatrix = new Float32Array([1,0,0,0, 0,1,0,0, 0,0,1,0, 0,0,0,1]);
+    const sensorAbs = new AbsoluteOrientationSensor({frequency: 60});
+    sensorAbs.onreading = () => sensorAbs.populateMatrix(modMatrix);
+    sensorAbs.start();
+
+    // Somewhere in rendering code, update vertex shader attribute for the model
+    gl.uniformMatrix4fv(modMatrixAttr, false, modMatrix);
+
+Orientation sensors enable various use cases, such as immersive gaming,
+augmented and virtual reality.
+
+For more information about motion sensors, advanced use cases and requirements,
+please check [motion sensors explainer](https://www.w3.org/TR/motion-sensors/)
+document.
+
+## Let’s code! {: #lets-code }
+
+The Generic Sensor API is very simple and easy-to-use! The Sensor interface has
+<code>[start()](https://w3c.github.io/sensors/#sensor-start)</code> and 
```

Comment:
No need to change, but this can be written as : 
```
[`start()`](https://www.example.com)
```
that way, you don't need to wrap in `<code>` blocks